public/index.php

```diff
@@ -34,7 +34,7 @@
 
 $app = require_once __DIR__.'/../bootstrap/app.php';
 
-// require_once __DIR__.'/../app/Constants.php';
+require_once __DIR__.'/../app/Constants.php';
```

Comment:
This change breaks the constants throughout the app for me. The `Constants.php` file loads via the `env()` helper method and at this point in the Laravel application lifecycle, the `env` values aren't available.